scripts/apps/relations/services/RelationsService.ts

```diff
@@ -3,12 +3,15 @@ import {IArticle} from 'superdesk-interfaces/Article';
 RelationsService.$inject = ['archiveService'];
 
 export function RelationsService(archiveService) {
-    this.getRelatedItems = function(item: IArticle) {
+    this.getRelatedItems = function(item: IArticle, mediaFields: IArticle) {
         if (!item.associations) {
             return [];
         }
 
-        const related = Object.values(item.associations);
+        const relatedWithoutMedia = _.pickBy(item.associations, (value, key) =>
```

Comment:
If an image is in media gallery it should not be validated but if it is in related items it should be validated.
I think here the check is only by item type and not by field type (media gallery and related items)

Reply:
fixed the issue